src/components/Link.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Linking, Text } from 'react-native';
+
+import {standardStyle} from '../assets/styles.js';
+
+class Link extends React.Component {
+
+  constructor(){
+      super();
+      this.openLink = this.openLink.bind(this);
+  }
+
+  static propTypes = {
+    href: React.PropTypes.string.isRequired,
+    text: React.PropTypes.string.isRequired,
+  }
+
+  render() {
+
+    const {text} = this.props;
+
+    return(
+      <Text style={styles.text} onPress={this.openLink}>
+		{text}
+      </Text>
+    );
+  }
+
+  openLink() {
```

Comment:
pro tip: if you write this as an arrow function, you don't have to bind it in the constructor